commons/src/main/java/org/infinispan/commons/marshall/UTF8Marshaller.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.commons.marshall;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import org.infinispan.commons.io.ByteBuffer;
+import org.infinispan.commons.io.ByteBufferImpl;
+
+public final class UTF8Marshaller extends AbstractMarshaller {
```

Comment:
9.0.x and master have https://github.com/infinispan/infinispan/blob/master/commons/src/main/java/org/infinispan/commons/marshall/StringMarshaller.java
So we should probably have a default constructor for that which uses UTF-8 (as well as backporting the class to 8.2.x)